packages/web-app-files/src/components/Search/List.vue

```diff
@@ -246,12 +275,18 @@ export default defineComponent({
 
     const availableTags = ref<Tag[]>([])
     const tagFilter = ref<VNodeRef>()
+    const mimeTypesFilter = ref<VNodeRef>()
     const tagParam = useRouteQuery('q_tags')
     const lastModifiedParam = useRouteQuery('q_lastModified')
+    const mimeTypesParam = useRouteQuery('q_mimeTypes')
     const fullTextParam = useRouteQuery('q_fullText')
 
     const displayFilter = computed(() => {
-      return unref(fullTextSearchEnabled) || unref(availableTags).length
+      return (
+        unref(fullTextSearchEnabled) ||
+        unref(availableTags).length ||
+        unref(modifiedDateCapability).enabled
+      )
```

Comment:
Shouldn't it also check for the `mimeTypeCapability`?

Reply:
wasnt sure because it should be always there but more checks are better ^^